data/hai/hai reveal 4 middle.txt

```diff
@@ -1114,46 +1140,190 @@ mission "Hai Reveal [A09] History of Appeasement"
+					goto "Katya warning"
+			label "elders why"
+			action
+				set "Hai Reveal [A09] History of Appeasement: label elders why"
+			`	The left Elder says, "Secrecy, mostly, to prevent a refugee crisis if the Hai become common knowledge. The bulk of the resources go to a few large pirate groups that control the anarchist worlds. We fund the development and protection of their planets, in return for them ensuring the wrong people don't find us."`
+			`	Another Elder adds, "Most of the deals are much smaller, though: when a pirate gang causes mischief, we pay them to stop. Hostages, theft, raids, and the like. Usually, the same gang never bothers us again." Teeneep perks up at the word "hostages," but doesn't speak.`
+				goto "elder questions"
+			label "elders secret"
+			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
+			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
+				goto diplomats
+			label "Katya warning"
+			`	The Elder on the right disagrees. "At this point, the Republic is too weakened to be a threat. I'm confident they will see reason, for now."`
+			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`
+			label diplomats
+			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable, but she remains silent as the three exit.`
+			choice
+				`	"That could have gone better."`
+				`	"It sounds like we've made some progress!"`
+			`	As you walk back to the diplomatic quarter, Sayari begins. "They didn't do anything, but at least they agreed to solve the problems when they become disasters."`
+			`	Teeneep objects, "We can't wait for a disaster before solving this!"`
+			`	Sayari agrees. "You're right, of course, in this matter. The Elders have ever struggled with change. They're not wise for this, but they were not elected to be so. They were elected to continue the long march of incremental development. Besides, you know far better than most exactly what challenges they contend with these past couple decades. More of them have died or retired than been unelected."`
+			choice
+				`	"What can we do to solve this, Sayari?"`
+				`	"Teeneep, I think we need quick action. You mentioned something back in the cave."`
+			`	Before she can respond, Xilin arrives with Batu in tow. They consult Sayari, and occasionally Teeneep, in a long discussion about the Navy knowing of the pirate troubles and willfully not intervening. Xilin sympathizes and suggests the Hai ask the Navy to rescue those captured.`
```

Comment:
Since it's established you're walking back to the diplomatic quarter just before this, I think it would feel more natural if you walked past where Xilin and Batu already were and they were to pull you into a room for this conversation.

Reply:
I just made that change.